integration_tests/suite/base/test_line_endpoint.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import re
+
+from hamcrest import (
+    assert_that,
+    contains_inanyorder,
+    has_entries,
+)
+
+from . import confd
+from ..helpers import fixtures
+from ..helpers.config import CONTEXT
+
+
+@fixtures.transport()
+@fixtures.sip_template()
+@fixtures.sip_template()
+@fixtures.registrar()
+def test_create_line_with_multiple_endpoints_error(
+    transport, template_1, template_2, registrar
+):
+    aor_section_options = [
+        ['@custom_variable', 'custom'],
+        ['qualify_frequency', '60'],
+        ['maximum_expiration', '3600'],
+        ['remove_existing', 'yes'],
+        ['max_contacts', '1'],
+    ]
+    auth_section_options = [['username', 'yiq8yej0'], ['password', 'yagq7x0w']]
+    endpoint_section_options = [
+        ['@custom_variable', 'custom'],
+        ['force_rport', 'yes'],
+        ['rewrite_contact', 'yes'],
+        ['callerid', '"Firstname Lastname" <100>'],
+    ]
+    identify_section_options = [
+        ['match', '54.172.60.0'],
+        ['match', '54.172.60.1'],
+        ['match', '54.172.60.2'],
+    ]
+    registration_section_options = [
+        ['client_uri', 'sip:peer@proxy.example.com'],
+        ['server_uri', 'sip:proxy.example.com'],
+        ['expiration', '120'],
+    ]
+    registration_outbound_auth_section_options = [
+        ['username', 'outbound-registration-username'],
+        ['password', 'outbound-registration-password'],
+    ]
+    outbound_auth_section_options = [
+        ['username', 'outbound-auth'],
+        ['password', 'outbound-password'],
+    ]
+
+    endpoint_sip_body = {
+        'name': "name",
+        'label': "label",
+        'aor_section_options': aor_section_options,
+        'auth_section_options': auth_section_options,
+        'endpoint_section_options': endpoint_section_options,
+        'identify_section_options': identify_section_options,
+        'registration_section_options': registration_section_options,
+        'registration_outbound_auth_section_options': registration_outbound_auth_section_options,
+        'outbound_auth_section_options': outbound_auth_section_options,
+        'transport': transport,
+        'templates': [template_1, template_2],
+        'asterisk_id': 'asterisk_id',
+    }
+    endpoint_sccp_body = {
+        'options': [
+            ["cid_name", "cid_name"],
+            ["cid_num", "cid_num"],
+            ["allow", "allow"],
+            ["disallow", "disallow"],
+        ],
+    }
```

Comment:
Any reason to have all this noise for this test?
```suggestion
def test_create_line_with_multiple_endpoints_error():
    endpoint_sip_body = {'name': "name"}
    endpoint_sccp_body = {'options': [["cid_name", "cid_name"]}
```